WikiWatershed/model-my-watershed
# Add layer toggles to analyze sidebar

## Overview

The MMW experience is greatly enhanced when you use the overlays. To call them out, we're adding related overlay toggles to each analyze tab (except animals).

#### Wireframe
![screen shot 2017-03-29 at 2 52 01 pm](https://cloud.githubusercontent.com/assets/7633670/24471397/0152b73a-1490-11e7-8d53-145243338e9f.png)

(Note in the above, Jeff gave sign off that the Turn On blue should actually match the link color in the rest of the app)

Connects #1745 
Connects #1776 

### Demo

![suj3f3azty](https://cloud.githubusercontent.com/assets/7633670/24471435/1ab87ef8-1490-11e7-96e9-ad74b0619c8e.gif)

### Notes

- The "Total Area" info at the top of each tab will move eventually, so don't worry about that not matching the wireframe

- For the water quality tab @jfrankl is working on a mockup of a collapsable list of layers (there are three: TP, TSS, TN). Will add a commit to style them accordingly once that's ready 

## Testing Instructions

 * Bundle
 * Test the layerpicker as its model has been refactored
     - DRB layers should be disabled when outside their extent
      - Go through each tab
 * Draw an AOI
 * Go through each tab analyze view and confirm you see the relevant layers for each
* The main layerpicker should be synced with the toggles
 * Pointsource should have a loading message while loading
